contracts/multisig-prover/src/events.rs

```diff
@@ -20,12 +18,12 @@ pub enum Event {
 impl From<Event> for cosmwasm_std::Event {
     fn from(other: Event) -> Self {
         match other {
-            Event::ProofUnderConstruction { proof_id } => {
-                cosmwasm_std::Event::new("proof_under_construction").add_attribute(
-                    "proof_id",
-                    to_string(&proof_id).expect("violated invariant: proof_id is not serializable"),
-                )
-            }
+            Event::ProofUnderConstruction {
+                multisig_session_id: proof_id,
+            } => cosmwasm_std::Event::new("proof_under_construction").add_attribute(
+                "proof_id",
```

Comment:
we should name the attribute "multisig_session_id" right?

Reply:
lol yes